example/feature_demo/demo_types.pb.gorm.go

```diff
@@ -254,10 +254,9 @@ type TypeWithIDWithAfterToPB interface {
 
 // MultiaccountTypeWithIDORM demonstrates the generated multi-account support
 type MultiaccountTypeWithIDORM struct {
-	AccountID                   string
-	Id                          uint64
-	SomeField                   string
-	MultiaccountTypeWithoutIDId uint32
```

Comment:
I'm assuming this field was removed in a previous commit but the file wasn't synced prior to merging?